ansible/roles/common-cartridge/defaults/main.yml

```diff
@@ -0,0 +1 @@
+EXTERNAL_SECRETS_POSTFIX: ""
```

Comment:
Do you realy intend to add `EXTERNAL_SECRETS_POSTFIX` to each Ansible role at the server repo even as you set it in your pr at dof_app_deploy

Reply:
That way this PR can be merged without changing anything and we only need to coordinate one merge (dof_app_deploy) when migrating. If we don't define a default, we would have to merge them at the same time.